packages/web/src/components/Alert/README.md

```diff
@@ -0,0 +1,14 @@
+# Alert
+
+Alert presents feedback or important information to users.
+
+```html
+<div class="Alert Alert--success">
```

Comment:
One more thing: could you please add `role="alert"` to all Alerts? Thank you!

Reply:
Fixed.